db/migrate/20250309121956_rename_ansible_tower_fqdn_to_api_url.rb

```diff
@@ -1,7 +1,7 @@
 class RenameAnsibleTowerFqdnToApiURL < ActiveRecord::Migration[7.0]
   def up
     Parameter.where(name: 'ansible_tower_fqdn').find_each do |parameter|
-      parameter.update(
+      parameter.update_columns(
```

Comment:
Can you please use [`update_all`](https://apidock.com/rails/v7.1.3.4/ActiveRecord/Relation/update_all) instead of loading each record into the memory?

Reply:
Something like:
```
    Parameter.where(name: 'ansible_tower_fqdn').update_all(
      "name = 'ansible_tower_api_url', value = 'https://' || value || '/api/controller/v2'"
    )
```
I didn't check if that works yet, just want to confirm before if that's what you mean.

Reply:
I see. Since the value is not stored as a string, but as a yaml, it becomes too complicated to update it from the DB. Let's keep the update_columns version then.